src/fiatconnect/saga.ts

```diff
@@ -1024,6 +1028,17 @@ export function* handleCreateFiatConnectTransfer(
         txHash: transactionHash,
       })
     )
+    if (transactionHash) {
```

Comment:
we won't have the transactionHash for transfers in, so it seems like this could be merged with the section above.
```suggestion
```

even if we changed the spec to have the provider give us transactionHash for transfers in, they'd need to debit the user's fiat account first before giving this, meaning we'd get the tx hash async after a potentially substantial delay. so I think it's very safe to assume that we'll only have transaction hash for transfers out.

Reply:
Updated it